x-pack/legacy/plugins/uptime/public/lib/adapters/framework/new_platform_adapter.tsx

```diff
@@ -33,10 +33,8 @@ export const getKibanaFrameworkAdapter = (
   core.chrome.getBreadcrumbs$().subscribe((nextBreadcrumbs?: ChromeBreadcrumb[]) => {
     breadcrumbs = nextBreadcrumbs || [];
   });
-  const { apm, infrastructure, logs } = getIntegratedAppAvailability(
-    capabilities,
-    INTEGRATED_SOLUTIONS
-  );
+  const availability = getIntegratedAppAvailability(capabilities, INTEGRATED_SOLUTIONS);
```

Comment:
We can remove the line below this if you want. One less symbol declared, and the destructuring can replace it:

```typescript
const { apm, infrastructure: metrics, logs } = getIntegratedAppAvailability(capabilities, INTEGRATED_SOLUTIONS);
```

We could even alias the declarations above so we don't need to have the field assignment in the props declaration below:

```typescript
const {
  apm: isApmAvailable,
  infrastructure: isMetricsAvailable,
  logs: isLogsAvailable
} = getIntegratedAppAvailability(capabilities, INTEGRATED_SOLUTIONS);
```

WDYT?